app/Website/src/Service/MailboxLayerService.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Website\Service;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use Illuminate\Support\Facades\Config;
+
+class MailboxLayerService
+{
+    protected Client $client;
+    protected string $baseUrl = 'http://apilayer.net/api';
+
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * @throws GuzzleException
+     */
+    public function check($email)
+    {
+        $accessKey = Config::get('mailboxlayer.access_key');
+        $response = $this->client->get("{$this->baseUrl}/bulk_check", [
```

Comment:
You can use the Http client that Laravel provides as an alternative but it's up to you, it's not required.